### ISSUE TITLE
[BUG] @self media image path is not updated when page is ordered

### ISSUE THREAD

**Issue by peca021** [getgrav/grav#2479]
When I change page order (Folder Numeric Prefix is enabled) @self media images path inside the .md file is not updated.

I have a page which name is Jobs and that page has ***05.*** prefix and my .md file for their inner page look like:

```
image:
    user/pages/05.jobs/data-scientist-machine-learning-developer-m-f-d/AczI4qr71un8S5d.svg:
        name: AczI4qr71un8S5d.svg
        type: image/svg+xml
        size: 3449
        path: user/pages/05.jobs/data-scientist-machine-learning-developer-m-f-d/AczI4qr71un8S5d.svg
```

When I move the page and change the order to ***06.jobs*** my path is still looking for the image inside folder ***05.jobs*** and I get error message 404 not found ...

Does anyone know how I can solve this problem? Thanks!

**Comment by mahagr** [comment]
Yup, this is a known issue when moving pages.

**Comment by peca021** [comment]
@mahagr can we expect a fix in some of the recent updates?

**Comment by rhukster** [comment]
This is not an easy fix, and a bit of an edge case for most users, so priority is a bit low. So no plans to fix this one right now...

**Comment by lufog** [comment]
The picture should be available by:
/jobs/data-scientist-machine-learning-developer-m-f-d/AczI4qr71un8S5d.svg

**Comment by mahagr** [comment]
I would change the image to be without any path (relative to the page), but as @rhukster said, it is not an easy fix with the current Page architecture. I do have plans to fix it, but it will not happen in Grav 1.6 nor with the current Page classes.

**Comment by mahagr** [comment]
This has now been implemented. Media inside the page doesn't have path anymore, which fixes the issue.
